### ISSUE TITLE
Find/create a nice graphic

### ISSUE THREAD

**Issue by aleksandar-todorovic** [aleksandar-todorovic/awesome-linux#23]
This awesome list could use a nice simple minimalistic graphic on top of it.

I have created one in a couple of minutes, but I'm far away from being satisfied with it, so if anyone wants to take a shot at creating something better, your help will be appreciated.

**Comment by davidak** [comment]
I also noticed that it don't look high quality. Maybe ask here for help:

http://opensourcedesign.net/jobs/
https://ura.design/

**Comment by aleksandar-todorovic** [comment]
Yup, unfortunately, that low quality graphic I've made kind of got stuck for way longer than I intended it to.

With that being said, I've contacted Ura Design and have interest in opening a request in Open source Design if that doesn't go well.

For the transparency, I can spend up to $50 trying to solve this. That's the amount I've offered Ura Design, and that's the amount I'm gonna offer in the Open Source Design application or whoever else manages to solve this.

The idea is:

1. See if the Ura Design accepts this job.
2. If not, I'll create a Bountysource bounty and post the job to Open Source Design as well.

If anyone wants to pitch in to Ura Design to make the compensation bigger, please, let me know.

If anyone wants to pitch in during the second stage (if it becomes a thing), there's gonna be a Bountysource bounty that you could pitch in.
